CoreTesting/nbproject/genfiles.properties

```diff
@@ -1,8 +0,0 @@
-build.xml.data.CRC32=5c21f638
-build.xml.script.CRC32=b5f9ba5a
-build.xml.stylesheet.CRC32=15ca8a54@2.79
```

Comment:
should this be submitted?

Reply:
I deleted that file and added it to .gitignore